chart/validator/templates/cleanup.yaml

```diff
@@ -86,23 +86,32 @@ metadata:
     "helm.sh/hook": pre-delete
 spec:
   template:
+    metadata:
+      labels:
+        app: validator-cleanup-job
     spec:
       restartPolicy: Never
       serviceAccountName: spectro-cleanup
       containers:
       - name: validator-cleanup
-        image: gcr.io/spectro-images-public/release/spectro-cleanup:1.1.0
+        image: gcr.io/spectro-images-public/release/spectro-cleanup:1.2.0
         command: ["/cleanup"]
         env:
         - name: CLEANUP_DELAY_SECONDS
-          value: {{ mul 10 (max 1 (len (required ".Values.plugins is required!" .Values.plugins))) | quote }}
+          value: {{ mul 15 (max 1 (len (required ".Values.plugins is required!" .Values.plugins))) | quote }}
```

Comment:
Since we now rely on the `FinalizeCleanup` endpoint to ensure fast cleanup, id make the argument that we should increase the `CLEANUP_DELAY_SECONDS` timeout to give us more buffer room. 

In reality, we should always be relying on the event to trigger the self destruction of spectro-cleanup, but in a future with more plugins, and on the off chance the validator is run on slow hardware, increasing the buffer would protect us from self destructing too soon

Reply:
Let's just pick a largish number then and not have a convoluted formula.

Reply:
bumped to 5 mins. that should definitely be long enough to clean up resources in the case that something goes wrong